pkgs/applications/misc/doomseeker/default.nix

```diff
@@ -19,9 +19,7 @@ stdenv.mkDerivation {
 
   enableParallelBuilding = true;
 
-  NIX_CFLAGS_COMPILE = stdenv.lib.optionalString stdenv.cc.isClang "-Wno-error=format-security";
```

Comment:
Is there a way I can test with clang here to see if we should still pass in `stdenv.cc` explicitly just for the `isClang`?  I scanned through the manual but didn't see the clang stdenv documentation.

Reply:
Derive it like
```
mkDerivationWith clangStdenv {
```

Reply:
The warning is triggered in macos related code, so it should be enough to s/stdenv.cc.isClang/stdenv.isDarwin/

Reply:
My motivation was wondering if there was some equivalent to `pkgsStatic.doomseeker` that I could try at the cmdline.  My first instinct when seeing these things is to wonder if the opt-out is bitrotted and no longer needed thanks to a version upgrade at some point.  

In this case, it looks like ofborg has confirmed this is still needed.  I'll do it properly by looking at the isClang attr of cc rather than replacing it with isDarwin.